docs/adapters/configuration.md

```diff
@@ -0,0 +1,105 @@
+## Goal
+Most simply and consistently across adapters, allow users to configure one or multiple connections in addition to a single path to the adapter module. 
+
+## Options and Problems
+
+### 1. Single connection object.
+- Eval: Simple, can include `path`, but does not allow for several connections/environments (e.g. prod/staging/dev).
+- Example: 
+```
+{
+  host: 'local',
+  port: 5432,
+  path: 'optional'
+}
+```
+
+### 2. An array of connection objects
+- Description: Each member of array has a unique `id` field so the user can indicate which connection to use.
```

Comment:
Let's add: "The first instance is treated as the default one, regardless of the name of its `id` field (ie: it is not necessary to name it `id: "default"`). A juttle program `read <adapter>` that doesn't specify `-id 'key'` will read from this first instance of the adapter config."